src/helpers.ts

```diff
@@ -25,6 +25,7 @@ export const isProdEnv =
   process.env.NODE_ENV === 'prod' ||
   !process.env.NODE_ENV ||
   (!isTestEnv && !isDevEnv);
+export const apiDocumentationUrl = process.env.API_DOCS_URL;
```

Comment:
Could you add an example and commented-out line and quick description in the `.env` file for this var? That's the main way we're "documenting" env vars right now.